server/mobile-services.js

```diff
@@ -268,31 +270,28 @@ const MobileSecurityService = {
     kind: 'MobileSecurityServiceApp'
   },
   getClientConfig: (namespace, appname, kubeclient) => {
-    const configmapName = `${appname}-security`;
-
-    return kubeclient.api.v1
+    const resourceName = `${appname}-security`;
+    return kubeclient.apis[mobileSecurityServiceCRD.spec.group].v1alpha1
       .namespaces(namespace)
-      .configmaps(configmapName)
+      .mobilesecurityserviceapps(resourceName)
       .get()
       .then(resp => resp.body)
-      .then(configmap => {
-        if (configmap) {
-          const sdkConfig = JSON.parse(configmap.data.SDKConfig);
-          const sdkConfigUrl = sdkConfig.url;
+      .then(mss =>
+        getServices(configPath).then(services => {
+          const mobileService = services.find(s => s.type === MOBILE_SECURITY_TYPE);
           return {
-            id: configmap.metadata.uid,
+            id: mss.metadata.uid,
             name: MOBILE_SECURITY_TYPE,
             type: MOBILE_SECURITY_TYPE,
-            url: sdkConfigUrl
+            url: mobileService.host
           };
-        }
-        return null;
-      })
+        })
+      )
       .catch(err => {
         if (err && err.statusCode && err.statusCode === 404) {
-          console.info(`Can not find configmap ${configmapName}`);
+          console.info(`\nCan not find Mobile Security Service App ${resourceName}\n`);
         } else {
-          console.warn(`Error when fetch configmap ${configmapName}`, err);
+          console.warn(`\nError when fetch Mobile Security Service App ${resourceName}\n`, err);
```

Comment:
I think for uniformity we should not have a newline for just this service. Logging improvements should come in the next phase.